glusterd2/commands/volumes/grouped-options.go

```diff
@@ -129,6 +129,12 @@ var defaultGroupOptions = map[string]*api.OptionGroup{
 			{Name: "features.shard", OnValue: "on"},
 			{Name: "user.cifs", OnValue: "off"}},
 		"Enable this profile, if the Gluster Volume is used to store virtual machines"},
+	"tls": {"tls",
+		[]api.VolumeOption{
+			{Name: "server.transport.socket.ssl-enabled", OnValue: "on"},
+			{Name: "client.transport.socket.ssl-enabled", OnValue: "on"},
```

Comment:
We should possibly try to get the `socket.ssl-*` options to `socket.tls-*`. Makes this more the option more consistent.